libtransmission/net.h

```diff
@@ -55,6 +55,13 @@ using tr_socket_t = int;
 #define sockerrno errno
 #endif
 
+/* macOS, FreeBSD, and OpenBSD all limit unix domain sockets to 104 characters. Linux is 108 so
+ * 104 is the limiting number here. However, our address adds "unix:" to the front so it must be
+ * 104 + 5 = 109 */
+#define TR_UNIX_ADDRSTRLEN 109
+
+#define TR_UNIX_SOCKET_PREFIX "unix:"
```

Comment:
- This doesn't need to be in the header

- In the source file, this should be a `static char constexpr TrUnixSocketPrefix[] = "unix:";`

Reply:
`TR_UNIX_SOCKET_PREFIX` is used in rpc-server.cc for copying the socket path minus the prefix length.